dashboard/app/views/pd/workshop_mailer/teacher_enrollment_reminder.html.haml

```diff
@@ -41,6 +41,10 @@
   = render partial: 'pre_survey'
 - elsif @workshop.subject != Pd::Workshop::COURSE_FACILITATOR
   %p
+    - if @is_first_pre_survey_email
+      Please take a moment to review the following logistical details about your workshop.
+    - else
+      If you have not already done so, please review the following logistical details about your workshop.
```

Comment:
Oh thank you for mentioning this I meant to leave a comment here. This will affect other emails as well but I checked with Dani on whether this should just be for the capstone or for all of them and she said we should unify it for all of them!